distribution/bwc/build.gradle

```diff
@@ -107,7 +107,7 @@ BuildParams.bwcVersions.forPreviousUnreleased { BwcVersions.UnreleasedVersionInf
     task checkoutBwcBranch() {
       dependsOn fetchLatest
       doLast {
-        String refspec = System.getProperty("bwc.refspec.${bwcBranch}") ?: System.getProperty("tests.bwc.refspec.${bwcBranch}") ?: "${remote}/${bwcBranch}"
+        def refspec = System.getProperty("bwc.refspec." + bwcBranch) ?: System.getProperty("tests.bwc.refspec." + bwcBranch) ?: "${remote}/${bwcBranch}"
```

Comment:
This is a result of gradle 6.5 updated groovy to 2.5.11 which results in this failure otherwise: https://gradle-enterprise.elastic.co/s/nv35erntpdz2e/failure